src/components/footer/FooterEntry.jsx

```diff
@@ -0,0 +1,13 @@
+function FooterEntry({ entry }) {
+  return (
+    <div className='flex flex-col gap-2'>
+        {Object.entries(entry).map(([key, value]) => (
```

Comment:
Damn I actually never knew you could do this. I've always just for looped through the object